### ISSUE TITLE
Implement Looks via OCIO

### ISSUE THREAD

**Issue by sobotka** [darbyjohnston/DJV#161]
Looks are not yet implemented in the pixel management chain.

https://twitter.com/djv_opensource/status/1196151370413273088?s=21

**Comment by darbyjohnston** [comment]
Pushing this to the next milestone...

**Comment by Rapheus** [comment]
Hello, Just discovered this thread,
Sry for posting this ticket https://github.com/darbyjohnston/DJV/issues/384

which is essentially the same request

**Comment by darbyjohnston** [comment]
Hi, no problem, I'll mark the other ticket as a duplicate and close it.

**Comment by sssssamvh** [comment]
Hi @darbyjohnston, any news on this?

**Comment by sobotka** [comment]
Congratulations, Darby!

**Comment by sobotka** [comment]
Yeah not sure about that. The whole Autodesk-Academy-ification of the otherwise stalwart OpenColorIO library is a bit of a tragedy.
